### DISCUSSION TITLE
Die Einstellung der Grundlast im DPL

### DISCUSSION THREAD

**Discussion by hoschiking** [hoylabs/OpenDTU-OnBattery#1388] [General]
der kleinstmögliche Wert ist 16W. Warum eigentlich?
Es ist eine tolle Funktion, die jetzt reingekommen ist, aber eine Einstellung auf 0 (also Abschaltung der Einspeisung)
wenn Stromzähler nicht verfügbar wäre wünschenswert.
Mir ist klar, dass diese Funktion hauptsächlich dafür gedacht ist, wenn man statisch eine bestimmte Leistung einspeisen möchte,
aber die kann man auch dafür nutzen, wenn der Stromzähler, warum auch immer, keine Werte mehr liefert, genau dann die Einspeisung abzuschalten, da ein undefinierter Zustand.

**Comment by spcqike** [comment]
Die Grundlast ist ein Fallback, wenn der Zähler nicht verfügbar ist. 

Wie sich die Grundlast bei ner Batterie auswirkt, weiß ich nicht. 
bei solarbetriebenen System gab es einfach keinen Grund, diese zu stoppen.

Vor der Einführung der Grundlast wurde der WR ausgeschaltet, wenn kein powermeter verfügbar war. Dazu gibt es meiner Meinung nach aber keinen Grund. eine gewisse und bekannte Grundlast hat doch jeder. Sei es der Router, drei Ladegeräte im Standby und der Fernseher. 

Bei Batteriensystemen könnte man das ganze ggf. noch mal hinterfragen. Auch wenn ich auch dort meine 200W gern nutzen würde, solange die Batterie es zulässt :) 

welchen Vorteil siehst du in der Abschaltung? Bzw, was ist das Problem an einer konstanten 16W Einspeisung? Ist deine Grundlast tatsächlich geringer?

**Reply by hoschiking** [reply]
Der größte Vorteil ist, dass es in der Praxis sehr oft vorkommt, dass es nicht nur eine Solaranlage gibt.
Gängig ist auch z.B. 2 Module per MPPT an den Speicher und 2 Module per WR direkt ans Netz.
In meinem Fall habe ich eine etwas größere Anlage auf dem Dach (7 Module a 400W), die speist direkt ein,
zusätzlich 2 Balkonkraftwerke mit je 800W, die speisen auch direkt per Micro-WR ein. Ich lade den Speicher nur AC-gestützt.
Gibt es tagsüber Solarüberschuß, fängt die OpdenDTUoB trotzdem einzuspeisen, falls Stromzähler nicht verfügbar,
statt die wertvolle Energie für dunkle Tageszeit bereitzuhalten.
Oder anders ausgedrückt, nur weil der Stromzähler nicht verfügbar ist, muss nicht automatisch Netzbezug anstehen.
Vorschlag: einfach (zusätzlich) einen Schalter einbauen, WR aus wenn Stromzähler nicht verfügbar.

**Reply by SW-Niko** [reply]
Hallo @hoschiking,
soweit ich das noch das im Kopf habe ist die Grundlast aus mehreren Gründen reingekommen.

- Reine Grundlast, wenn man keinen Stromzähler hat oder keinen benötigt
- Übergangsweise auf Grundlast gehen wenn der Stromzähler vorübergehend keine Daten liefert. (WLAN Ausfall, Router reboot, usw)

Diese beiden Funktionalitäten sind also genau das Gegenteil das du benötigst.
Der kleinste Wert muss etwas höher als das "minimum power limit" sein. Vermutlich damit der Inverter nicht schwingt.

![grafik](https://github.com/user-attachments/assets/651ce45b-8f84-4823-8479-7132319eecbe)

Ein zusätzlicher Schalter, der dann auch die Grundlast ausblendet wäre sicher eine Lösung. Alternativ könnte man in das Feld auch 0W eingeben um anzuzeigen das der Inverter abgeschaltet werden soll.

**Reply by hoschiking** [reply]
So ist es, auch dein Vorschlag wäre genauso gut, aber keine Möglichkeit zu haben, die Einspeisung auszuschalten wenn Stromzähler nicht verfügbar, finde ich leider einen Rückschritt.

**Comment by schlimmchen** [comment]
> Vorschlag: einfach (zusätzlich) einen Schalter einbauen, WR aus wenn Stromzähler nicht verfügbar.

Meine Standardreaktion bei sowas: "Auf gar keinen Fall gibt es dafür einen extra Schalter" :wink:

Der Grund, dass das min hier auf ein Watt mehr als das untere Leistungslimit festgelegt wurde (von mir) ist ganz einfach: Ich dachte, es sei sinnvoll, diesen Wert zu zwingen auf die Range (minimales Limit, maximales Limit). Das Maximum hab ich allerdings gar nicht (versucht) als `max` für dieses Eingabefeld zu definieren?!

Jedenfalls ist das Unsinn, denn @hoschiking's Argument, dass man sich auch wünschen können sollte, dass man nichts produziert, wenn der PowerMeter ausfällt, ist absolut nachvollziehbar.
